code/modules/events/disease_outbreak.dm

```diff
@@ -1,3 +1,6 @@
+#define MAJOR_INTO_MODERATE TRUE  //If true folds die major diseases/symptoms into moderate ones
```

Comment:
the compiler doesnt like this. this makes always true statements. I would prefer to just not have the option to change this. we can always just revert it if its bad

Reply:
I put it into an #ifdef which seems like the proper way to do it,

I mean, I can get rid of it being changeable, I just think still having that logic there might be useful for the virology expansion/rework in the future.

Reply:
I would rather avoid using `ifdef`

Reply:
Alright. Will rework it tonight.

Reply:
This should be it then, unless I missed something.
Works in testing and the list in the debugger is correct.